testing/grafana/APKBUILD

```diff
@@ -0,0 +1,49 @@
+# Contributor: Konstantin Kulikov <k.kulikov2@gmail.com>
+# Maintainer: Konstantin Kulikov <k.kulikov2@gmail.com>
+pkgname=grafana
+pkgver=6.2.5
+pkgrel=0
+_commit=6082d1982 # git rev-parse --short HEAD
+_stamp=1561485379 # git --no-pager show -s --format=%ct
+pkgdesc="Open source, feature rich metrics dashboard and graph editor"
+url="https://grafana.com"
+arch="all"
+license="Apache-2.0"
+makedepends="go nodejs yarn python"
```

Comment:
can python3 be used ?

Reply:
python dep was optional, removed

Reply:
ok, it seems to be optional for x86_64, but required for x86. I'll remove x86 support then

Reply:
Please remove x86 support, we are also working on removing python2

Reply:
done